Helper/PlaceHelper.php

```diff
@@ -83,6 +85,45 @@ public function getPlaceList( Location $location, $contentTypes, $languages = ar
     }
 
     /**
+     * Retrieve all objects related to place $content.
+     *
+     * @param \eZ\Publish\Core\Repository\ContentService $contentService
+     * @param \eZ\Publish\API\Repository\Values\Content\Content $content
+     * @return array
+     */
+    public function getRelatedObjects( ContentService $contentService, Content $content )
+    {
+        $relatedObjects = array();
+
+        $relations = $contentService->loadRelations( $content->versionInfo );
+        foreach ( $relations as $rel )
+        {
+            $relatedObjects[] = $contentService->loadContent( $rel->destinationContentInfo->id );
+        }
+
+        return $relatedObjects;
+    }
+
+    /**
+     * Returns first image from related objects array.
+     *
+     * @param \eZ\Publish\Core\Repository\ContentService $contentService
+     * @param \eZ\Publish\API\Repository\Values\Content\Location $location
+     * @return \eZ\Publish\API\Repository\Values\Content\Content
+     */
+    public function getImageFromRelatedObjects( ContentService $contentService, Location $location )
+    {
+        $content = $contentService->loadContentByContentInfo( $location->getContentInfo() );
+
+        $relatedObjects = $this->getRelatedObjects(
```

Comment:
Overkill when you plan to only return one in the end, please assume database (including API calls that implicitly make one) is on another planet in terms of time it takes for an answer.